packages/web-react/src/components/FileUploader/FileUploaderList.tsx

```diff
@@ -14,17 +14,18 @@ const FileUploaderList = (props: SpiritFileUploaderListProps) => {
 
   const renderAttachments = useMemo(() => {
     const fileArray = Array.from(fileQueue, (entry) => {
-      return { key: entry[0], file: entry[1] };
+      return { key: entry[0], file: entry[1].file, meta: entry[1].meta };
```

Comment:
I think we can use explicit return here.

Reply:
changed to implicit return